commit b2222a797cfd65ee44b224ea90f3317725396253
Author: Mohamed Khalfella <mohamed.khalfella@joyent.com>
Date:   2019-09-06T20:11:49+00:00 (4 weeks ago)
    
    OS-7974 bhyve does not persist nic_driver
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/src/vm/node_modules/proptable.js b/src/vm/node_modules/proptable.js
index d9b7fdec..9042627d 100644
--- a/src/vm/node_modules/proptable.js
+++ b/src/vm/node_modules/proptable.js
@@ -1156,6 +1156,7 @@ exports.properties = {
     }, nic_driver: {
         payload: {
             allowed: {
+                'bhyve': ['create', 'receive', 'update'],
                 'kvm': ['create', 'receive', 'update']
             },
             type: 'string'
